.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: java
+before_cache:
```

Comment:
Do we want to add https://github.com/hmcts/service-auth-provider-java-client/blob/master/.travis.yml#L2-L4

Reply:
no, that would block publishing, also its a silly thing to add imo.
It stops travis CI from running on branches other than master.